nextstrain_profiles/nextstrain-gisaid-21L/builds.yaml

```diff
@@ -0,0 +1,648 @@
+auspice_json_prefix: ncov_gisaid_21L
+
+# Define custom rules for pre- or post-standard workflow processing of data.
+custom_rules:
+  - workflow/snakemake_rules/export_for_nextstrain.smk
+
+# These parameters are only used by the `export_for_nextstrain` rule and shouldn't need to be modified.
+# To modify the s3 _source_ bucket, specify this directly in the `inputs` section of the config.
+# P.S. These are intentionally set as top-level keys as this allows command-line overrides.
+S3_DST_BUCKET: "nextstrain-ncov-private"
+S3_DST_COMPRESSION: "xz"
+S3_DST_ORIGINS: ["gisaid"]
+upload:
+  - build-files
+
+# Deploy and Slack options are related to Nextstrain live builds and don't need to be modified for local builds
+deploy_url: s3://nextstrain-data
+slack_token: ~
+slack_channel: "#ncov-gisaid-updates"
+
+genes: ["ORF1a", "ORF1b", "S", "ORF3a", "E", "M", "ORF6", "ORF7a", "ORF7b", "ORF8", "N", "ORF9b"]
+use_nextalign: true
+include_hcov19_prefix: True
+
+files:
+  include: "nextstrain_profiles/nextstrain-gisaid-21L/include.txt"
+  description: "nextstrain_profiles/nextstrain-gisaid-21L/nextstrain_description.md"
+
+# Note: unaligned sequences are provided as "aligned" sequences to avoid an initial full-DB alignment
+# as we re-align everything after subsampling.
+inputs:
+  - name: gisaid
+    metadata: "s3://nextstrain-ncov-private/metadata.tsv.gz"
+    aligned: "s3://nextstrain-ncov-private/sequences.fasta.xz"
+    skip_sanitize_metadata: true
+  - name: references
+    metadata: "data/references_metadata.tsv"
+    aligned: "data/references_sequences.fasta"
+    skip_sanitize_metadata: true
+
+# Define locations for which builds should be created.
+# For each build we specify a subsampling scheme via an explicit key.
+# These subsampling schemes are defined at the bottom of this file.
+# (They override the defaults)
+# North America and Oceania are subsampled at the "division" level
+# Africa, Asia, Europe and South America are subsampled at the "country" level
+#
+# Auspice config is specified in rule auspice_config in export_for_nextstrain.smk
+builds:
+  reference:
+    subsampling_scheme: nextstrain_reference
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with clade-focused subsampling
+  global_2m:
+    subsampling_scheme: nextstrain_global_2m
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused globally over the past 2 months
+  global_6m:
+    subsampling_scheme: nextstrain_global_6m
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused globally over the past 6 months
+  global_all-time:
+    subsampling_scheme: nextstrain_global_all-time
+    title: Evolution SARS-CoV-2 relative to clade 21L reference virus with subsampling focused globally since pandemic start
```

Comment:
GitHub doesn't [show this file as a copy + change](https://tsibley.net/paste/2023-03-30Nk3QbSco-tmp-28232-LkyTxc.html), which is unfortunate, so it's hard to see here, but the 1m timespans are missing due to timing of when they were added and this PR. We should probably add them back?